deployment/helm/templates/coap-core.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.coapcore.enabled }}
+apiVersion: akri.sh/v0
+kind: Configuration
+metadata:
+  name: {{ .Values.coapcore.name }}
+spec:
+  protocol:
+    coapcore:
+      {{- if .Values.coapcore.ipAddresses}}
+      ipAddresses: 
+        {{- toYaml .Values.coapcore.ipAddresses | nindent 8 }}
+      {{- else }}
```

Comment:
I dont think this is necessary since `ipAddresses` defaults to `[]` if nothing is specified